src/EntryPoints/Rest/GetAllMappingsApi.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\WikibaseRDF\EntryPoints\Rest;
+
+use MediaWiki\MediaWikiServices;
+use MediaWiki\Rest\SimpleHandler;
+use ProfessionalWiki\WikibaseRDF\WikibaseRdfExtension;
+use stdClass;
+
+class GetAllMappingsApi extends SimpleHandler {
+
+	/**
+	 * @psalm-suppress all
+	 * @return array<string, mixed>
+	 */
+	public function run(): array {
+		$lb = MediaWikiServices::getInstance()->getDBLoadBalancer();
+		$dbr = $lb->getConnectionRef( DB_REPLICA );
+
+		$res = $dbr->newSelectQueryBuilder()
+			->select( [
+				// Actual fields
+				'text.old_text',
+				'page.page_title',
+				// TODO: debug fields
+				'page.page_id',
+				'text.old_id',
+			] )
+			->from( 'text' )
+			->join( 'page', conds: 'text.old_id=page.page_latest' )
```

Comment:
It looks like we don't need the `content` table, but we do need the `page` table to get the entity id (title). These joins end up returning what I'm looking for, but I'm not sure if I'm somehow abusing the ids being joined.

Reply:
I suspect this is not correct. 

In this screenshot from yesterday, my last two edits (revisions 6 and 7) were to the RDF slot on Q2. At that point, page_latest will be 7 for Q2. But as soon as you edit the main slot there, it will become 8, which will correspond to a new wikitext entry in the `text` table.

Reply:
You're right. I didn't check when doing non-mapping updates. That breaks the query.